app/models/concerns/applicable_to_nations.rb

```diff
@@ -0,0 +1,30 @@
+module ApplicableToNations
+  extend ActiveSupport::Concern
+
+  included do
+    validate :applicable_to_at_least_one_nation
```

Comment:
So it's a [ActiveModel::Validator](http://guides.rubyonrails.org/active_record_validations.html#custom-validators), which gets picked up by `ActiveRecord` and makes it possible to do: `validates uri: true`, which you can see an example of here: https://github.com/alphagov/whitehall/blob/master/app/models/organisation.rb#L147